apps/condo/domains/news/tasks/notifyResidentsAboutNewsItem.js

```diff
@@ -60,7 +61,7 @@ async function sendNotifications (context, newsItem, taskId) {
 
     checkSendingPossibility(newsItem)
 
-    const scopes = await NewsItemScope.getAll(context, { newsItem: { id: newsItem.id } })
+    const scopes = await find('NewsItemScope', { newsItem: { id: newsItem.id } })
```

Comment:
`find` and other direct utils (`getById`, `getByCondition`) and others does not filter out deleted objects by default. You must add it by yourself